tests/network/localnet/conftest.py

```diff
@@ -125,13 +165,18 @@ def vm_localnet_2(
     cudn_localnet: libcudn.ClusterUserDefinedNetwork,
     unprivileged_client: DynamicClient,
 ) -> Generator[BaseVirtualMachine]:
+    networks = [Network(name=LOCALNET_BR_EX_NETWORK, multus=Multus(networkName=cudn_localnet.name))]
+    interfaces = [Interface(name=LOCALNET_BR_EX_NETWORK, bridge={})]
+    network_data = cloudinit.NetworkData(
+        ethernets={PRIMARY_INTERFACE_NAME: cloudinit.EthernetDevice(addresses=[next(ipv4_localnet_address_pool)])}
+    )
```

Comment:
This looks similar to the previous one [1], please follow the same pattern.|

Ref: https://github.com/RedHatQE/openshift-virtualization-tests/pull/2370#discussion_r2507756854